src/trigger.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <span ref="trigger" :class="[addClass]"><slot></slot></span>
+    <span ref="trigger" :class="[addClass || (this.trigger === 'click' ? 'click-trigger' : 'other-trigger')]"><slot></slot></span>
```

Comment:
`addClass` by convention allows the author to append additional classes to the component, but it seems to not be doing that here? :/

Reply:
Yeah, it is overriding. Should I create another pr to revert this?

Reply:
> Should I create another pr to revert this?

Yes please do. If that means that you can no longer remove the dash, you can also see if there's alternative ways of getting rid of dotted lines (e.g. maybe introduce a `disable-dash` attribute or a `style` attribute to turn it off).